packages/cli/src/commands/server/launchDebugger.ts

```diff
@@ -69,6 +68,7 @@ function getChromeAppName(): string {
 }
 
 function launchChrome(url: string) {
+  // @ts-ignore open's callback argument definition is not defined in open
   open(url, {app: [getChromeAppName()]}, err => {
```

Comment:
Maybe using ąsync/await syntax would help with the typings? Based on the readme this is what they recommend